control-board/src/bin/control/main.rs

```diff
@@ -185,15 +185,21 @@ async fn main(_spawner: embassy_executor::Spawner) {
     info!("id: {}", robot_id);
     shell_indicator.set(robot_id);
 
-    let wifi_network = if dip5.is_high() & dip6.is_high() {
-        WifiNetwork::Team
-    } else if dip5.is_low() & dip6.is_high() {
-        WifiNetwork::CompMain
-    } else if dip5.is_high() & dip6.is_low() {
-        WifiNetwork::CompPractice
-    } else {
-        WifiNetwork::Team
-    };
+    let wifi_network = WifiNetwork::Team;
+    // let wifi_network = if dip5.is_high() & dip6.is_high() {
```

Comment:
It's a note for me to re-integrate these features basically. We need a big clean up and it may happen with a hardware rev to address the broken rotary encoder.